mobile-workspace/latest/using/index.md

```diff
@@ -232,3 +232,21 @@ You can add attachments to a task.
     > **Note:** You may need to scroll further down the page to see all the attachments.  
 
 ![Add attachment]({% link mobile-workspace/images/add-attachment.png %})
+
+## Share custom URLs
+
+You can share custom Workspace links provided by the URL Schema functionality. The link can go directly to a particular folder or files, for example **Personal Files**, **Custom Folder**, or **File Preview**.
+
+1. Tap on the custom Workspace URL:
+
+    * If you're using an iOS device, you're prompted to open the link with a browser.
+    * If you're using an Android device, the Mobile Workspace application launches. You may need to log in, if prompted.
+
+2. Select a browser to view the content (for iOS):
+
+    * On iOS, you're asked to open the link using the Alfresco Mobile application (Alfresco Mobile Workspace) or download Mobile Workspace from the App Store.
+    * On Android, you're redirected to the Play Store.

```

Comment:
It should be like below in the case of Android.

On Android, you're asked to open the link using the Alfresco Mobile application (Alfresco Mobile Workspace). On tap, It will open the Mobile App or redirect to the Play Store if the app is not installed.